experimental/StandardFiniteFields/test/runtests.jl

```diff
@@ -0,0 +1,44 @@
+using Test
+using Oscar
+GAP.Packages.load("StandardFF")
+
+function MyPoly(p,n)
+    return defining_polynomial(standard_finite_field(p,n))
+end
+
+function GAPPoly(p,n)
+    poly = GAP.evalstr( "StandardFiniteField(" * string(p) * ", " * string(n) * ")!.DefiningPolynomial!.CoefficientsOfUnivariatePolynomial")
+    poly = polynomial(GF(p), map(x -> GF(p)(x), GAP.gap_to_julia(Vector{GAP.FFE}, poly)))
```

Comment:
```suggestion
    poly = polynomial(GF(p), map(GF(p), Vector{GAP.FFE}(poly)))
```
The use of `gap_to_julia` is not type-safe.